src/ui/room_exits.ui

```diff
@@ -1920,7 +1920,7 @@
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="toolTip">
-         <string>&lt;p&gt;Click on an item to edit/change it, to DELETE a Special Exit set its Exit Room ID to zero.&lt;/p&gt;</string>
+         <string>&lt;p&gt;Click on an item to edit/change it; to delete a Special Exit, select it and press the keyboard Delete key OR set its Exit Room ID to less than 1 OR clear the name/command entry.&lt;/p&gt;</string>
```

Comment:
Keep it at 0, even though setting it to -23 could work as well, that sentence had too much info already.
Switch the options, mention the most simple one first.
Split the first idea off to make it shorter.
Replace more CAPS LOCKED words.
Use common "roomID" syntax.
```suggestion
         <string>&lt;p&gt;Click on an item to edit/change it. To delete a Special Exit, either set its roomID to 0, or select it and press the keyboard Delete key, or clear the name/command entry.&lt;/p&gt;</string>
```

Reply:
> Keep it at 0, even though setting it to -23 could work as well, that sentence had too much info already.

Sorry but I disagree, as it also warns that any value less than one **will** always cause exit deletion and not just that single value of zero - and in English, when referring to a number with an absolute value less than 100 it is usual to spell the number out in words...

> Switch the options, mention the most simple one first.

The simplest one *is* click on the exit and press delete - *and* it works on multiple exits at a time (but I wasn't going to mention *that*)!

> Split the first idea off to make it shorter.

Not sure about that as the previous point also alludes to click-and-drag to select several exits at a time. I guess it does improve the readability if it is a separate sentence. 

> Replace more CAPS LOCKED words.

Well, I thought that using OR helped to separate the alternatives, but will try with an "... either: xxxx; or yyyyy; or zzzz." form.

> Use common "roomID" syntax.

Yeah, I am making that change throughout - even though it offends my sensibilities - as we have previously had that discussion about *id*entity (noun) / *I*(dentiy) *D*(ocument) --> *ID* (abreviation) / *Id*entity (English, capitalised, proper noun) and the democratically derived result was ID...